docs/topics/assembly_deploying-runtime-application-to-openshift.adoc

```diff
@@ -0,0 +1,13 @@
+[id='deploying-runtime-application-to-openshift_{context}']
+= Deploying {runtime} application to OpenShift
+
+ifdef::built-for-vertx[]
+To deploy an {runtime} application to OpenShift, you must configure the application's `pom.xml` file. You can then deploy the application using Fabric8 Maven launcher.
```

Comment:
Agreed. Nice solution. 

- Changed to "your" {runtime} application.
- Rewrote the sentence to avoid Saxon genitive: 
 "To deploy your {runtime} application to OpenShift, configure the pom.xml file in your application and then deploy using Fabric8 Maven plugin."
- Changed *Fabric8 Maven launcher* to *Fabric8 Maven plugin*.